rpm/nemo-qml-plugin-thumbnailer-qt5.spec

```diff
@@ -11,6 +11,10 @@ BuildRequires:  pkgconfig(Qt5Qml)
 BuildRequires:  pkgconfig(Qt5Quick)
 BuildRequires:  pkgconfig(mlite5)
 BuildRequires:  sailfish-qdoc-template
+Requires: thumbnaild
+Provides: nemo-qml-plugin-thumbnailer-qt5-video
+Obsoletes: nemo-qml-plugin-thumbnailer-qt5-libav
```

Comment:
Please no unversioned obsoletes.

Reply:
Was pondering that in a way a version is also wrong, e.g. if I just put the current latest there this should still obsolete a theoretical new version too until the setup is adjusted again. In practice the new version doesn't happen so with that could add it just to please the build system, but would be nice to have some real rationale, so far haven't found such.

Reply:
It should be the last version that exists.

Reply:
Sure it gets rid of the warning but still: why should that be commonly done other than to get rid of rpmlint warning? :)

But maybe that question could be raised on more generic level, so I'll just add it here.

Reply:
One reasoning is that reintroducing a package is difficult if there is no version in obsoletes. Of course sometimes that's not a concern but usually you don't plan beforehand to reintroduce something if you are obsoleting it.

Reply:
Could be but then again on this sort of situations reintroduction of a package likely needs changes also on the places that was doing the obsoletion. Having old version of that and new version of the reintroduced package would be also wrong.

Reply:
Regarding the version number, I think a good choice is to take the last real version and add `+git1`, or bump up the last number, even if the package doesn't use this `+something` notation. Usually this is needed when we also do provides (so `Provides: xxx = yyy+git1` and `Obsoletes: xxx < yyy+git1`), which is not done here though. And regardless of provides I would perhaps use exclusive comparison when the version number is not an existing version of the package.

Reply:
Using now < 0.1.0, rationale being that if theoretically anything is adjusted ever again on that package it would warrant bump from 0.0.x to 0.1.0 at least.